src/sage/quadratic_forms/ternary_qf.py

```diff
@@ -522,8 +513,8 @@ def reciprocal(self):
         """
         Return the reciprocal quadratic form associated to the given form.
 
-        This is defined as the multiple of the primitive adjoint with the same
-        content as the given form.
+        This is defined as the multiple of the :meth:`primitive` :meth:`adjoint` with the same
+        :meth:`content` as the given form.
 
```

Comment:
Why `:meth:` here?

Reply:
All of these are not just mathematical terms but also the names of methods that compute that

Reply:
I should check this. I cannot agree at this moment.

Reply:
I saw the full docstring, and understand why you said so. But I still think these are bad changes. The author used the terms "primitive adjoint" and "content" as mathematical terms. The author used "defined" as mathematical definition.
It is bad that "primitive adjoint" is rendered "primitive() adjoint()"!

Reply:
OK, I agree.

Reply:
Reverted.